src/sage/categories/drinfeld_modules.py

```diff
@@ -23,10 +23,12 @@
 from sage.categories.homsets import Homsets
 from sage.misc.functional import log
 from sage.misc.latex import latex
+from sage.misc.lazy_import import lazy_import
 from sage.rings.integer import Integer
-from sage.rings.polynomial.ore_polynomial_ring import OrePolynomialRing
-from sage.rings.polynomial.polynomial_ring import PolynomialRing_general
-from sage.rings.ring_extension import RingExtension_generic
+
+lazy_import('sage.rings.ring_extension', 'RingExtension_generic')
```

Comment:
Very minor detail: shouldn't this line be just after the last lazy import (alphabetical order)?

Reply:
Thanks, fixed.